examples/pipeline_config/example_pipeline.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
nit: move this to a pipelines directory.
This way we aren't mixing CDAP JSON configs w/ tf HCL configs.